extra-devel/cc-tool/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME="cc-tool"
+PKGSEC=devel
+PKGDES="A developing tool for Texas Instruments' CC Debugger"
```

Comment:
"Support library and tools for Texas Instruments CC Debugger"

Reply:
`cc-tool` is only a flash util. it doesnt provide any libraries.